### ISSUE TITLE
Refetch Events not working after moving from CroudTech/vue-fullcalendar

### ISSUE THREAD

**Issue by bbashy** [fullcalendar/fullcalendar-vue#110]
Link to reproduce: https://codesandbox.io/s/bold-tu-zvkmm?file=/src/DemoApp.vue

Related to #58 I guess.

I use axios to send requests via the `eventSources` and I have a select input to filter the data on the calendar.
It was all working fine in https://github.com/CroudTech/vue-fullcalendar

![](https://d.code.im/kic2IR184Pm6xO5GoViLQbgxw15haQYnV3F2Op0HRM.jpg)

No new network requests are happening even though the event is firing.

I use this in my actual project
```js
eventSources(date, callback) {
    let startDate = moment(date.start).format('YYYY-MM-DD');
    let endDate = moment(date.end).format('YYYY-MM-DD');

    if (! this.category) {
        axios.get(`/api/periodicals?start=${startDate}&end=${endDate}`).then(response => {
            callback(response.data.data);
        });
        return;
    }

    axios.get(`/api/periodicals/${this.category.value}?start=${startDate}&end=${endDate}`).then(response => {
        callback(response.data.data);
    });
},
```

**Comment by acerix** [comment]
Your demo is not loading events from a source or calling the API correctly, it works as expected by changing that:

https://codesandbox.io/s/divine-dawn-l866b?file=/src/DemoApp.vue

https://fullcalendar.io/docs/vue#calendar-api

**Comment by bbashy** [comment]
Your codesandbox link showed no changes but I figured it out in the end from your reply.

```
let calendarApi = this.$refs.fullCalendar.getApi();
calendarApi.refetchEvents();
```

I'm not used to dealing with methods directly from Vue as most packages have listeners and events to do changes.

Thanks for the help. I got confused from the previous package and also the issue I linked. I was trying to do `getApi().$emit('refetch-events');` thinking I couldn't use refetchEvents() on the getApi for some reason.

Hope it helps someone else stuck upgrading.
